### DISCUSSION TITLE
Listed: Support image from File for a Super note type

### DISCUSSION THREAD

**Discussion by mdubrois** [standardnotes/forum#3613] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
With the Super note type, when I use an image from the `File` that image is not displayed on the Blog.  

**Describe the solution you'd like**
I would like to see the image on the Blog.  
I guess, we can assume that if I share a post that links to a File, the File is shared too. 

**Describe alternatives you've considered**
Nope

**Additional context**
Nothing

**Comment by Donnnno** [comment]
Would love to see this feature too!
